data/json/items/book/cooking.json

```diff
@@ -436,20 +439,21 @@
   {
     "id": "textbook_chemistry",
     "type": "BOOK",
-    "name": "chemistry textbook",
-    "description": "A college textbook on chemistry.",
+    "name": "Chemistry Textbook",
+    "name_plural": "Chemistry Textbooks",
+    "description": "This college textbook of practical organic chemistry explains the basic applications of chemistry to everyday living.",
```

Comment:
This doesn't sound like any college organic chem text I've ever seen. I'd say "This college organic chemistry textbook drily explains in excruciating detail dozens of chemical reaction mechanisms both in theory and in practice."

Reply:
```suggestion
    "description": "This college organic chemistry textbook dryly explains in excruciating detail dozens of chemical reaction mechanisms both in theory and in practice.",
```

Reply:
This might not be relevant, but this specific edit was inspired after reviewing a half dozen  chemistry textbooks published between 1945 and 1965. It is trivial and can be eliminated without second thoughts. My intent was to provide amusing copy.

I believe the original was _"A college textbook on chemistry."_